samples/sdl_ttf/main.c

```diff
@@ -0,0 +1,241 @@
+/*
+  Copyright (C) 1997-2016 Sam Lantinga <slouken@libsdl.org>
+
+  This software is provided 'as-is', without any express or implied
+  warranty.  In no event will the authors be held liable for any damages
+  arising from the use of this software.
+
+  Permission is granted to anyone to use this software for any purpose,
+  including commercial applications, and to alter it and redistribute it
+  freely.
+*/
+/* Simple program:  Create a native window and attach an SDL renderer */
+#include <xboxrt/debug.h>
+#include <pbkit/pbkit.h>
+#include <hal/xbox.h>
+#include <hal/video.h>
+#include "stdio.h"
+#include "string.h"
+#include <SDL.h>
+#include <SDL_ttf.h>
+
+int _exit(int x)
+{
+    debugPrint("_exit called\n");
+    debugPrint( " SDL_Error: %s\n", SDL_GetError() );
```

Comment:
This feels more like an `_fatal_error` than `_exit`.

Also, why is it `_` prefixed?

Reply:
SDL seems to be the culprit here. When `_exit` was removed a compilation error occured, "symbol __exit not found" or something to that effect.

Reply:
This sounds like a bug in our SDL then?

Someone should look into this and send a separate PR. We shouldn't have to sacrifice our sample code quality for bugs elsewhere.

Reply:
Quite possibly.